src/arweaveTransport/arweaveTransport.ts

```diff
@@ -0,0 +1,16 @@
+import { arweave as arweaveInstance } from './arweave';
```

Comment:
Let's introduce new modules/classes only when we really need to. In this specific case I would rather add a helper function that would check the tx status directly to arweave instance. I believe having each small piece of functionality in a separate class is confusing. At least in this case I don't see any reason for that. Please correct me if I am wrong.